tensorflow_model_optimization/g3doc/guide/clustering/clustering_with_keras.ipynb

```diff
@@ -0,0 +1,510 @@
+{
```

Comment:
<p>Would break this down into two code blocks to distinguish the somewhat boilerplate code (the helper function) from the actual evaluation calls. </p><p> </p><p>From my comment above, I'd move the TF evaluation down here so that the side by side is clearer.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>